lefthook.yml

```diff
@@ -17,6 +17,10 @@ pre-commit:
       root: "frontend/"
       glob: "*.scss"
       run: ./node_modules/.bin/stylelint  --fix {staged_files} && git add {staged_files}
+    eslint-func:
```

Comment:
Oooh thank you for spotting and fixing this gap! Should make developing on the function much better going forward 😄